src/components/Input.tsx

```diff
@@ -50,7 +50,7 @@ const Input = forwardRef<HTMLInputElement, Props>(
           type={type}
           inputMode={type === 'number' ? 'numeric' : 'text'}
           placeholder={placeholder}
-          value={value}
+          defaultValue={value}
```

Comment:
@taehee-kim-sherpas could you specify the details? I just tested it with react and it works for me

Reply:
Tested from example/react app - that is with React 19.1.0

Reply:
ok, there seems to be something wrong, you are right